### PR TITLE
Proofread the documentation

### PR DESCRIPTION
**Pull request by patiencedaur**
* Proofread the English language
* Provide a more human space name
* Provide the correct translation of the term "space"

Resolves https://github.com/tarantool/doc/issues/2545

### CODE REVIEW

File: doc/guide.ru.rst [hunk 1]
```diff
@@ -68,17 +68,17 @@ Tarantool поддерживает три типа полей: ``STR``, ``NUM``
...
 
 Как видно из примера, все значения были преобразованы в Python-типы в соответствии со схемой.
 
 Кортеж Tarantool может содержать произвольное количество полей.
-Если какие-то поля не объявлены в схеме, то ``default_type`` будет использован для конвертации.
+Если какие-то поля не объявлены в схеме, то для конвертации будет использован ``default_type``.
 
 Поля с "сырыми" байтами следует использовать, если приложение работает с
-двоичными данными (например, изображения или python-объекты, сохраненные с помощью ``picke``).
+двоичными данными (например, с изображениями или python-объектами, сохраненными с помощью ``pickle``).
```
Comment (oleg-jukovec) [inline doc/guide.ru.rst:81]:
```suggestion
двоичными данными (например, с изображениями или Python-объектами, сохраненными с помощью ``pickle``).
```

File: tarantool/connection.py [hunk 2]
```diff
@@ -332,13 +332,13 @@ def connect_unix(self):
...
         '''
-        Wrap an existing socket with SSL socket.
+        Wrap an existing socket with an SSL socket.
 
         :raise: SslError
         :raise: `ssl.SSLError`
         '''
         if not is_ssl_supported:
-            raise SslError("SSL is unsupported by the python.")
+            raise SslError("Your version of python doesn't support SSL")
```
Comment (oleg-jukovec) [inline tarantool/connection.py:341]:
```suggestion
            raise SslError("Your version of Python doesn't support SSL")
```

**Review by oleg-jukovec — CHANGES_REQUESTED** [review]
Thank you for the great work! Could you please resolve the notes, please?

File: doc/index.ru.rst [hunk 3]
```diff
@@ -56,9 +56,9 @@
 
 
 .. _`Tarantool`:
-.. _`Tarantool homepage`: http://tarantool.org
-.. _`Tarantool/Box User Guide`: http://tarantool.org/tarantool_user_guide.html
-.. _`Mail.Ru`: http://mail.ru
+.. _`Tarantool homepage`: https://tarantool.io
+.. _`Документация Tarantool`: https://www.tarantool.io/en/doc/latest/
```
Comment (oleg-jukovec) [inline doc/index.ru.rst:60]: We can refer to a russian version of docs here:
```suggestion
.. _`Документация Tarantool`: https://www.tarantool.io/ru/doc/latest/
```

File: tarantool/connection.py [hunk 4]
```diff
@@ -763,33 +765,33 @@ def upsert(self, space_name, tuple_value, op_list, **kwargs):
...
-
         * `+` for addition (values must be numeric)
         * `-` for subtraction (values must be numeric)
         * `&` for bitwise AND (values must be unsigned numeric)
         * `|` for bitwise OR (values must be unsigned numeric)
         * `^` for bitwise XOR (values must be unsigned numeric)
-        * `:` for string splice (you must provide `offset`, `count` and `value`
-          for this operation)
+        * `:` for string splice (you must provide `offset`, `count`,
+           and `value` for this operation)
```
Comment (oleg-jukovec) [inline tarantool/connection.py:794]:
```suggestion
          and `value` for this operation)
```
The markup is broken due to the extra space.
list

**Review by oleg-jukovec — APPROVED** [review]
Thank you for the fixes! LGTM after solving two small problems.
You could squash the commits into one if you want.

File: tarantool/connection.py (hunk 4 above)
Comment (patiencedaur) [reply tarantool/connection.py:794]: Thank you!

File: doc/index.ru.rst (hunk 3 above)
Comment (patiencedaur) [reply doc/index.ru.rst:60]: I was thinking about this, but we currently don't support Russian documentation. It looks like a patchwork of Russian and English text and may be hard to read for this reason. Are you sure you'd prefer the Russian link?

Comment (patiencedaur) [reply doc/index.ru.rst:60]: Also, I want to squash the commits but it isn't enabled for this repository:
image

Comment (oleg-jukovec) [reply doc/index.ru.rst:60]: > I was thinking about this, but we currently don't support Russian documentation. It looks like a patchwork of Russian and English text and may be hard to read for this reason. Are you sure you'd prefer the Russian link?

Well there is no problem. Let it stay 'en'.

**Comment by oleg-jukovec** [comment]
> Also, I want to squash the commits but it isn't enabled for this repository:

I meant something like this:
https://www.baeldung.com/ops/git-squash-commits#1-squash-the-last-x-commits

**Comment by patiencedaur** [comment]
> I meant something like this:
> https://www.baeldung.com/ops/git-squash-commits#1-squash-the-last-x-commits

@oleg-jukovec Done.

**Comment by oleg-jukovec** [comment]
> > I meant something like this:
> > https://www.baeldung.com/ops/git-squash-commits#1-squash-the-last-x-commits
> 
> @oleg-jukovec Done.

Thank you! To make everything perfect, let's synchronize the commit message with the format used in the repository:

```
prefix: subject

description
```

It can be something like that:
```
docs: proofread

* Proofread the English and Russian language
* Provide a more human space name
* Provide the correct translation of the term "space"
```

**Comment by oleg-jukovec** [comment]
Thank you!

**Pull request merged**